contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
+    default: "/opt/stackstorm/packs/examples/actions/workflows/print_text.sl"
+    type: string
+  inputs:
```

Comment:
Yeah, that's something we could do - we could write a little action metadata generator script which generates StackStorm action metadata for a particular CloudSlang flow.

Retrieving the inputs dynamically during run time is also possible, but that's something we should avoid since it doesn't fit well into how StackStorm works and breaks many things such as parameter introspection.

First approach does require a bit more work if the flow file changes (user needs to re-run the metadata generator script), but it fits better into the StackStorm model.

Circling back to the original question - alright, I will make a change which generates inputs dictionary from the action parameters and hopefully get it done and merged into master by EOD today.